evm/src/cpu/kernel/tests/core/access_lists.rs

```diff
@@ -1,88 +1,211 @@
 use std::collections::HashSet;
 
 use anyhow::Result;
-use ethereum_types::{Address, U256};
+use ethereum_types::{Address, H160, U256};
+use hashbrown::hash_map::rayon::IntoParIter;
 use rand::{thread_rng, Rng};
 
 use crate::cpu::kernel::aggregator::KERNEL;
 use crate::cpu::kernel::constants::global_metadata::GlobalMetadata::{
     AccessedAddressesLen, AccessedStorageKeysLen,
 };
 use crate::cpu::kernel::interpreter::Interpreter;
-use crate::memory::segments::Segment::{AccessedAddresses, AccessedStorageKeys};
+use crate::memory::segments::Segment::{self, AccessedAddresses, AccessedStorageKeys};
+use crate::memory::segments::SEGMENT_SCALING_FACTOR;
 use crate::witness::memory::MemoryAddress;
 
+#[test]
+fn test_init_access_lists() -> Result<()> {
+    let init_label = KERNEL.global_labels["init_access_lists"];
+
+    // Test for address already in list.
```

Comment:
Is this comment accurate? It seems like the test is testing that the initial state of the access lists is what we expect? Maybe "Test for the initial state of the access list in the kernel"